views/agreement.hbs

```diff
@@ -312,12 +325,73 @@ input[type="submit"] {
     </div>
    </fieldset>
 
-  <p>(To change this information, e.g. to change the Primary Contact, notify the <a href="https://whatwg.org/sg-agreement#steering-group">Steering Group</a> using <a href="agreement-update">the Workstream update interface</a>.)</p>
+  <p>(To change this information, e.g., to change the Primary Contact, notify the <a href="https://whatwg.org/sg-agreement#steering-group">Steering Group</a> using <a href="agreement-update">the Workstream update interface</a>.)</p>
 
 
   </div>
  </details>
 
+ <script>
```

Comment:
The contents of this element can be merged into the next `script` element, no?

Reply:
Sure looks like, I'll try making the edit.